framework/commerce/product/use-price.tsx

```diff
@@ -10,7 +10,9 @@ export function formatPrice({
   currencyCode: string
   locale: string
 }) {
-  const formatCurrency = new Intl.NumberFormat(locale, {
+  console.log('Use-Price Locale', { locale })
+  // TODO: YB Change locale here
+  const formatCurrency = new Intl.NumberFormat('en-US', {
```

Comment:
For some reason I was getting the following error: `Incorrect locale information provided`